ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/ConfigEnricher.scala

```diff
@@ -13,7 +12,7 @@ object ConfigEnricher {
 
   def enrichStreamConfig(
       streamConfig: StreamConfig,
-      submissionSummary: Option[CommandSubmitter.SubmissionSummary],
+      submissionSummary: Option[SubmissionStepResult],
```

Comment:
You have changed the type, but that also changes the semantic of the field. Shouldn't it be reflected in the parameter name?

Reply:
Renamed